### ISSUE TITLE
app errors

### ISSUE THREAD

**Issue by davegariepy** [spencercarli/react-native-meteor-boilerplate#29]
Hi, thanks very much for sharing this boilerplate.  I followed your directions to install, but haven't been able to get the app working on the iOS simulator.

When the simulator opens up, it is all red with a note at the top: 
this._nativeModule.addListener is not a function. (In 'this._nativeModule.addListener(eventType)','this._nativeModule.addListener' is undefined)

I have been using meteor successfully for a while now and it is still working in my other projects. Also, following the react-native installation guide, I was able to get the 'AwesomeProject' working from the Testing Installation section.

When I did npm install from RNApp I also got this:
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated cross-spawn-async@2.2.4: cross-spawn no longer requires a build toolchain, use it instead!
npm WARN prefer global babel@5.8.38 should be installed with -g

**Comment by davegariepy** [comment]
ok, so now I got it to build. The simulator shows "RNApp" then a loading spinner, but it stays on the loading spinner.

**Comment by MadeInMoon** [comment]
Hi ddddave,

Did you execute `meteor` by cli in the MeteorApp folder?

**Comment by smooJitter** [comment]
@ddddave,  it could be that your meteor server isn't running.  Any meteor server running on localhost://3000 (I believe the boilerplate watches on port 3000).

**Comment by spencercarli** [comment]
Just wanted to echo above, make sure that the meteor app is running. If you're using android (even in an emulator) you'll need to change localhost to your computer's IP address

**Comment by jefflau** [comment]
Hi guys,

I have the same problem as above. I'm just trying to get the stock boilerplate running. I have the meteor server running on a different terminal window and have run npm install. Then I ran the iOS app using xcode

``` bash
2016-07-13 12:05:05.793 [error][tid:com.facebook.react.JavaScript] this._nativeModule.addListener is not a function. (In 'this._nativeModule.addListener(eventType)', 'this._nativeModule.addListener' is undefined)
2016-07-13 12:05:05.797 [fatal][tid:com.facebook.react.RCTExceptionsManagerQueue] Unhandled JS Exception: this._nativeModule.addListener is not a function. (In 'this._nativeModule.addListener(eventType)', 'this._nativeModule.addListener' is undefined)
2016-07-13 12:05:05.810 [error][tid:com.facebook.react.JavaScript] Requiring module "329", which threw an exception.
2016-07-13 12:05:05.811 [fatal][tid:com.facebook.react.RCTExceptionsManagerQueue] Unhandled JS Exception: Requiring module "329", which threw an exception.
2016-07-13 12:05:05.813 [warn][tid:com.facebook.react.JavaScript] Unable to symbolicate stack trace: this._nativeModule.addListener is not a function. (In 'this._nativeModule.addListener(eventType)', 'this._nativeModule.addListener' is undefined)
2016-07-13 12:05:05.813 [warn][tid:com.facebook.react.JavaScript] Unable to symbolicate stack trace: this._nativeModule.addListener is not a function. (In 'this._nativeModule.addListener(eventType)', 'this._nativeModule.addListener' is undefined)
2016-07-13 12:05:07.654 [error][tid:com.facebook.react.JavaScript] Module AppRegistry is not a registered callable module.
2016-07-13 12:05:07.655 [warn][tid:com.facebook.react.JavaScript] Unable to symbolicate stack trace: this._nativeModule.addListener is not a function. (In 'this._nativeModule.addListener(eventType)', 'this._nativeModule.addListener' is undefined)
2016-07-13 12:05:07.655 [fatal][tid:com.facebook.react.RCTExceptionsManagerQueue] Unhandled JS Exception: Module AppRegistry is not a registered callable module.
```

**Comment by spencercarli** [comment]
@jefflau thanks for the showing the errors! Looks like you might be using symlinks somewhere? If so, [symlinks are not currently supported in React Native](https://github.com/facebook/react-native/issues/637) (due to watchman).

**Comment by jefflau** [comment]
I haven't changed anything in the actual project. It's all stock from a clone. Is there another way symlinks could affect it?

**Comment by spencercarli** [comment]
@jefflau maybe try `rm -rf node_modules && npm install` (in the RN app). It seems like React Native might not have installed correctly

**Comment by spencercarli** [comment]
Hoping this is fixed after merging #39. Let me know if not!
